gitpod-io/gitpod
# [ws-daemon] Preserve file extended attributes

**How to test:**
- Create a workspace using https://github.com/rfay/cap-add-demo as source
- Wait until the console appears and show something like this:

![Screenshot from 2021-04-01 22-57-00](https://user-images.githubusercontent.com/161571/113371862-b3935f00-933d-11eb-831b-c5b290840e54.png)

- Stop the workspace
- Open the workspace
- The console should display the same output
- Optionally, use port-forward with minio, download the backup, decompress the file and run `getcap` against the `nc.traditional` file

fixes #3721

**Note:** any other approach that doesn't require new capabilities is welcome
